moveit_experimental/package.xml

```diff
@@ -0,0 +1,87 @@
+<package>
+  <name>moveit_experimental</name>
+  <version>0.7.2</version>
+  <description>Experimental packages for moveit.</description>
+  <author email="isucan@gmail.com">Ioan Sucan</author>
+  <author email="robot.moveit@gmail.com">Sachin Chitta</author>
+  <author email="acorn.pooley@sri.com">Acorn Pooley</author>
+  <author email="jnicho@swri.org">Jorge Nicho</author>
+
+  <maintainer email="robot.moveit@gmail.com">Sachin Chitta</maintainer>
+  <maintainer email="mferguson@fetchrobotics.com">Michael Ferguson</maintainer>
+  <maintainer email="chitt@live.in">Chittaranjan Srinivas Swaminathan</maintainer>
+
+  <license>BSD</license>  
+  <url type="website">http://moveit.ros.org</url>
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>pkg-config</buildtool_depend>
+
+  <build_depend>pluginlib</build_depend>
+  <build_depend>libconsole-bridge-dev</build_depend>
+  <build_depend>liburdfdom-dev</build_depend>
+  <build_depend>liburdfdom-headers-dev</build_depend>
+  <build_depend>random_numbers</build_depend>
+  <build_depend>shape_tools</build_depend>
+  <build_depend>octomap_msgs</build_depend>  
+  <build_depend>moveit_msgs</build_depend>
+  <build_depend>actionlib_msgs</build_depend>
+  <build_depend>geometry_msgs</build_depend>
+  <build_depend>sensor_msgs</build_depend>
+  <build_depend>shape_msgs</build_depend>
+  <build_depend>std_msgs</build_depend>
+  <build_depend>trajectory_msgs</build_depend>
+  <build_depend>visualization_msgs</build_depend>   
+  <build_depend>octomap</build_depend>
+  <build_depend>srdfdom</build_depend>
```

Comment:
just a note - even though the srdfdom dependency is likely pulled in from another dep, if it is used directly in this package than it should be listed in the CMakeLists